infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -741,8 +741,8 @@ private boolean processDiscoveredTopology(TopologyDescriptor topology, boolean u
 
         // If a cluster descriptor is not found, this node does not belong to any cluster in the topology
         // wait for updates to the topology config to start, if this cluster ever becomes part of the topology
-        log.warn("Node[{}/{}] does not belong to any cluster provided by the discovery service, topology={}",
-                localEndpoint, localNodeId, topology);
+        log.warn("Node[{}/{}] does not belong to any cluster provided by the discovery service",
```

Comment:
printing the topology as part of the same message is really important for readability purposes.. In the same message you are able to verify why this node is not part of any cluster, i.e., what is the provider giving you.